### ISSUE TITLE
[Feature request] Fruit and Veggie data from USDA

### ISSUE THREAD

**Issue by huffstler** [Flunzmas/seasoncalendar#115]
**Is your feature request related to a problem? Please describe.**
Related to #47, there's a USDA website for America. While it doesn't seem like it has a LOT of information, some is better than none, right?

**Describe the solution you'd like**
I'd like to use the information provided from the USDA to start the North America (US?) region in this app. I have no idea if the information is licensed or considered "open" or not. At this point.

**Describe alternatives you've considered**
I'll continue looking for other sites providing similar information.

**Additional context**

Link to site is here: 

https://snaped.fns.usda.gov/seasonal-produce-guide

**Comment by matzebond** [comment]
I get "Access Denied" errors on several usda.gov pages (including your link) so I can't judge the amount/quality of the data.
We would very much like to include the region "North America" and any help is appreciated!
Could you contact the organization and ask them for permission to reuse their data. I guess this is a formality since it's a government organization who usually make their work publicly available.
Maybe you could ask for a data export of some kind. This would make integration into our database easier.

OT:  Did you find the app on f-droid?

**Comment by huffstler** [comment]
> OT: Did you find the app on f-droid?

Yup! Updated my repos and saw a new update for the app. I've been thinking about in-season veg to use for cooking like this recently, so it was a very nice coincidence :)

Wow, I wouldn't think the pages are region locked.... Interesting.

I can try to get in touch with them. What's game-plan if they never end up getting back to me?

Also, I found this site which seems to be a little more filled out with information: 

https://www.seasonalfoodguide.org/

I'll see if I can contact them as well.

**Comment by matzebond** [comment]
> What's game-plan if they never end up getting back to me?

We will see then :-)

> Also, I found this site which seems to be a little more filled out with information:
> 
> https://www.seasonalfoodguide.org/

Interesting site they do have their own app though.

**Comment by matzebond** [comment]
@huffstler did you get a response? :-)

**Comment by huffstler** [comment]
@matzebond Sorry about the delay here, life is continuously getting in the way 😅

I've sent an email, and am waiting for a response. I did a little more digging into data behind USDA food information and found the following pages:
- https://fdc.nal.usda.gov/api-guide.html
- https://fdc.nal.usda.gov/faq.html

USDA Seems to have an API to access, AS WELL AS the following statement on their API page: 

![image](https://user-images.githubusercontent.com/9092204/161054643-bffeb2c8-f841-4b52-bc45-b9d4aa67452a.png)

**Comment by matzebond** [comment]
I had looked at the https://snaped.fns.usda.gov/seasonal-produce-guide. It is very rough. We should nonetheless use it as a starting point for the USA region and hope that the users to complete the data with the new edit feature.

The USDA API makes nutrition data available but seasonality is missing as it seems to be more focused on processed foods.

**Comment by huffstler** [comment]
I have not had a soul respond to me at this point :( I'm sure they have better things to worry about than me asking weird questions

**Comment by huffstler** [comment]
Found a different food related database from the front page of Hacker News!

https://world.openfoodfacts.org/data

https://news.ycombinator.com/item?id=31898218
